android/fastlane/report.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+  <testsuite name="fastlane.lanes">
+    
+    
+    
+      
```

Comment:
Do you need to track this file?
Check this out: https://docs.fastlane.tools/best-practices/source-control/